### PR TITLE
Konstantinos / [FE Deriv.com squad] Careers > Locations > Change links for 'View open positions in <Location Name>' buttons for each Location.

### PR DESCRIPTION
**Pull request by konstantinosG-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [x] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/careers/_model/_locations/_locations.ts [hunk 1]
```diff
@@ -32,6 +32,7 @@ import SingaporeFlagIcon from 'images/svg/careers/flag_singapore.svg'
 
 export const cyberjaya: LocationsType = {
     name: 'cyberjaya',
+    jobs_name: 'Cyberjaya',
```
Comment (mitra-deriv) [inline src/pages/careers/_model/_locations/_locations.ts:35]: Hi @konstantinosG-deriv just a suggestion, Do you think we can use a better name for this? Like `location_name`  or `job_location`. 🙏

**Review by mitra-deriv — APPROVED** [review]

File: src/pages/careers/_model/_locations/_locations.ts [hunk 2]
```diff
@@ -66,6 +67,7 @@ export const cyberjaya: LocationsType = {
 
 export const malta: LocationsType = {
     name: 'malta',
+    jobs_name: 'Birkirkara',
```
Comment (habib-deriv) [inline src/pages/careers/_model/_locations/_locations.ts:70]: As @mitra-deriv suggested, let's change the key names
```suggestion
    job_location: 'Birkirkara',
```

File: src/common/constants.ts [hunk 3]
```diff
@@ -120,6 +120,7 @@ export const p2p_huawei_appgallery_url = 'https://appgallery.huawei.com/#/app/C1
 export const pushwoosh_app_code = 'DD293-35A19'
 export const sample_rate = 25
 export const zoho_url = 'https://deriv.zohorecruit.eu'
+export const zoho_locations_url = 'https://deriv.zohorecruit.eu/jobs/'
```
Comment (habib-deriv) [inline src/common/constants.ts:123]:
```suggestion
export const zoho_jobs_url = 'https://deriv.zohorecruit.eu/jobs/'
```

File: src/pages/careers/_model/_locations/_locations.ts (hunk 1 above)
Comment (konstantinosG-deriv) [reply src/pages/careers/_model/_locations/_locations.ts]: Ok good idea, thanks 
👍

**Review by mitra-deriv — APPROVED** [review]

**Review by habib-deriv — APPROVED** [review]

**Pull request merged**